2015-12-14 16:59:49 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User1', 18) [42102-190]
2015-12-14 17:20:51 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User0', 0) [42102-190]
2015-12-14 17:20:51 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User1', 1) [42102-190]
2015-12-14 17:20:51 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User2', 4) [42102-190]
2015-12-14 17:20:51 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User3', 9) [42102-190]
2015-12-14 17:20:51 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User4', 16) [42102-190]
2015-12-14 17:20:51 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User5', 25) [42102-190]
2015-12-14 17:20:51 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User6', 36) [42102-190]
2015-12-14 17:20:51 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User7', 49) [42102-190]
2015-12-14 17:20:51 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User8', 64) [42102-190]
2015-12-14 17:20:51 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User9', 81) [42102-190]
2015-12-14 17:20:51 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User10', 100) [42102-190]
2015-12-14 17:20:51 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User11', 121) [42102-190]
2015-12-14 17:20:51 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User12', 144) [42102-190]
2015-12-14 17:20:51 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User13', 169) [42102-190]
2015-12-14 17:20:51 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User14', 196) [42102-190]
2015-12-14 17:25:01 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User0', 0) [42102-190]
2015-12-14 17:25:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User1', 1) [42102-190]
2015-12-14 17:25:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User2', 4) [42102-190]
2015-12-14 17:25:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User3', 9) [42102-190]
2015-12-14 17:25:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User4', 16) [42102-190]
2015-12-14 17:25:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User5', 25) [42102-190]
2015-12-14 17:25:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User6', 36) [42102-190]
2015-12-14 17:25:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User7', 49) [42102-190]
2015-12-14 17:25:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User8', 64) [42102-190]
2015-12-14 17:25:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User9', 81) [42102-190]
2015-12-14 17:25:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User10', 100) [42102-190]
2015-12-14 17:25:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User11', 121) [42102-190]
2015-12-14 17:25:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User12', 144) [42102-190]
2015-12-14 17:25:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User13', 169) [42102-190]
2015-12-14 17:25:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User14', 196) [42102-190]
2015-12-14 17:26:04 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User0', 0) [42102-190]
2015-12-14 17:26:04 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User1', 1) [42102-190]
2015-12-14 17:26:04 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User2', 4) [42102-190]
2015-12-14 17:26:04 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User3', 9) [42102-190]
2015-12-14 17:26:04 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User4', 16) [42102-190]
2015-12-14 17:26:04 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User5', 25) [42102-190]
2015-12-14 17:26:04 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User6', 36) [42102-190]
2015-12-14 17:26:04 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User7', 49) [42102-190]
2015-12-14 17:26:04 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User8', 64) [42102-190]
2015-12-14 17:26:04 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User9', 81) [42102-190]
2015-12-14 17:26:04 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User10', 100) [42102-190]
2015-12-14 17:26:04 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User11', 121) [42102-190]
2015-12-14 17:26:04 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User12', 144) [42102-190]
2015-12-14 17:26:04 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User13', 169) [42102-190]
2015-12-14 17:26:04 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User14', 196) [42102-190]
2015-12-14 17:26:25 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User0', 0) [42102-190]
2015-12-14 17:26:25 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User1', 1) [42102-190]
2015-12-14 17:26:25 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User2', 4) [42102-190]
2015-12-14 17:26:25 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User3', 9) [42102-190]
2015-12-14 17:26:25 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User4', 16) [42102-190]
2015-12-14 17:26:25 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User5', 25) [42102-190]
2015-12-14 17:26:25 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User6', 36) [42102-190]
2015-12-14 17:26:25 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User7', 49) [42102-190]
2015-12-14 17:26:25 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User8', 64) [42102-190]
2015-12-14 17:26:25 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User9', 81) [42102-190]
2015-12-14 17:26:25 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User10', 100) [42102-190]
2015-12-14 17:26:25 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User11', 121) [42102-190]
2015-12-14 17:26:25 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User12', 144) [42102-190]
2015-12-14 17:26:25 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User13', 169) [42102-190]
2015-12-14 17:26:25 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User14', 196) [42102-190]
2015-12-14 17:26:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User0', 0) [42102-190]
2015-12-14 17:26:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User1', 1) [42102-190]
2015-12-14 17:26:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User2', 4) [42102-190]
2015-12-14 17:26:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User3', 9) [42102-190]
2015-12-14 17:26:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User4', 16) [42102-190]
2015-12-14 17:26:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User5', 25) [42102-190]
2015-12-14 17:26:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User6', 36) [42102-190]
2015-12-14 17:26:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User7', 49) [42102-190]
2015-12-14 17:26:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User8', 64) [42102-190]
2015-12-14 17:26:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User9', 81) [42102-190]
2015-12-14 17:26:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User10', 100) [42102-190]
2015-12-14 17:26:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User11', 121) [42102-190]
2015-12-14 17:26:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User12', 144) [42102-190]
2015-12-14 17:26:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User13', 169) [42102-190]
2015-12-14 17:26:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
INSERT INTO users (name, age) VALUES ('User14', 196) [42102-190]
2015-12-14 17:36:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Недопустимое значение "4" для параметра "parameterIndex"
Invalid value "4" for parameter "parameterIndex" [90008-190]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.getInvalidValueException(DbException.java:228)
	at org.h2.jdbc.JdbcPreparedStatement.setParameter(JdbcPreparedStatement.java:1382)
	at org.h2.jdbc.JdbcPreparedStatement.setString(JdbcPreparedStatement.java:352)
	at DatabaseServiceTest.testInsert(DatabaseServiceTest.java:34)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:117)
	at com.intellij.rt.execution.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:234)
	at com.intellij.rt.execution.junit.JUnitStarter.main(JUnitStarter.java:74)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:144)
2015-12-14 19:22:42 jdbc[156]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "SELECT * FROM USERS WHERE NAME[*]== ? ORDER BY AGE "
Syntax error in SQL statement "SELECT * FROM USERS WHERE NAME[*]== ? ORDER BY AGE "; SQL statement:
SELECT * FROM users WHERE name== ? ORDER BY age [42000-190]
2015-12-14 19:28:04 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "TABLE" не найдена
Table "TABLE" not found; SQL statement:
INSERT INTO TABLE users (name, age) VALUES ('Alla', 198) [42102-190]
2015-12-14 19:28:22 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "SELECT * FROM USERS WHERE NAME[*]== ? ORDER BY AGE "
Syntax error in SQL statement "SELECT * FROM USERS WHERE NAME[*]== ? ORDER BY AGE "; SQL statement:
SELECT * FROM users WHERE name== ? ORDER BY age [42000-190]
2015-12-14 19:28:54 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "SELECT * FROM USERS WHERE NAME[*]== ? ORDER BY AGE "
Syntax error in SQL statement "SELECT * FROM USERS WHERE NAME[*]== ? ORDER BY AGE "; SQL statement:
SELECT * FROM users WHERE name== ? ORDER BY age [42000-190]
2015-12-14 19:30:45 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "SELECT * FROM USERS WHERE NAME[*]== ? ORDER BY AGE "
Syntax error in SQL statement "SELECT * FROM USERS WHERE NAME[*]== ? ORDER BY AGE "; SQL statement:
SELECT * FROM users WHERE name== ? ORDER BY age [42000-190]
2015-12-14 19:31:14 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "SELECT * FROM USERS WHERE NAME[*]== ? ORDER BY AGE "
Syntax error in SQL statement "SELECT * FROM USERS WHERE NAME[*]== ? ORDER BY AGE "; SQL statement:
SELECT * FROM users WHERE name== ? ORDER BY age [42000-190]
2015-12-14 19:31:40 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "SELECT * FROM USERS WHERE NAME[*]== ? ORDER BY AGE "
Syntax error in SQL statement "SELECT * FROM USERS WHERE NAME[*]== ? ORDER BY AGE "; SQL statement:
SELECT * FROM users WHERE name== ? ORDER BY age [42000-190]
2015-12-14 19:31:56 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "SELECT * FROM USERS WHERE NAME[*]== ? ORDER BY AGE "
Syntax error in SQL statement "SELECT * FROM USERS WHERE NAME[*]== ? ORDER BY AGE "; SQL statement:
SELECT * FROM users WHERE name== ? ORDER BY age [42000-190]
2015-12-14 19:36:40 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "SELECT * FROM USERS WHERE NAME[*]== ? ORDER BY AGE "
Syntax error in SQL statement "SELECT * FROM USERS WHERE NAME[*]== ? ORDER BY AGE "; SQL statement:
SELECT * FROM users WHERE name== ? ORDER BY age [42000-190]
2015-12-14 19:41:00 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "SELECT * FROM USERS WHERE NAME[*]== ? ORDER BY AGE "
Syntax error in SQL statement "SELECT * FROM USERS WHERE NAME[*]== ? ORDER BY AGE "; SQL statement:
SELECT * FROM users WHERE name== ? ORDER BY age [42000-190]
2015-12-14 19:41:35 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "SELECT * FROM USERS WHERE NAME[*]== ? ORDER BY AGE "
Syntax error in SQL statement "SELECT * FROM USERS WHERE NAME[*]== ? ORDER BY AGE "; SQL statement:
SELECT * FROM users WHERE name== ? ORDER BY age [42000-190]
2015-12-14 19:41:57 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "SELECT * FROM USERS WHERE NAME[*]== ? ORDER BY AGE "
Syntax error in SQL statement "SELECT * FROM USERS WHERE NAME[*]== ? ORDER BY AGE "; SQL statement:
SELECT * FROM users WHERE name== ? ORDER BY age [42000-190]
2015-12-16 01:26:55 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Метод разрешен только для запросов. Используйте execute или executeUpdate вместо executeQuery
Method is only allowed for a query. Use execute or executeUpdate instead of executeQuery; SQL statement:
DELETE FROM users WHERE name= ? [90002-190]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.command.Prepared.query(Prepared.java:212)
	at org.h2.command.CommandContainer.query(CommandContainer.java:90)
	at org.h2.command.Command.executeQuery(Command.java:196)
	at org.h2.jdbc.JdbcPreparedStatement.executeQuery(JdbcPreparedStatement.java:108)
	at dbwork.DatabaseService.update(DatabaseService.java:238)
	at DatabaseServiceTest.testUpdate(DatabaseServiceTest.java:141)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:117)
	at com.intellij.rt.execution.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:234)
	at com.intellij.rt.execution.junit.JUnitStarter.main(JUnitStarter.java:74)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:144)
2015-12-16 03:53:33 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" уже существует
Table "USERS" already exists; SQL statement:
CREATE TABLE users (name varchar(255), age int);  [42101-190]
2015-12-16 03:53:33 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
DROP TABLE users [42102-190]
2015-12-16 03:53:33 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
DROP TABLE users [42102-190]
2015-12-16 03:53:33 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
DROP TABLE users [42102-190]
2015-12-16 03:53:33 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
DROP TABLE users [42102-190]
2015-12-16 03:54:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
DROP TABLE users [42102-190]
2015-12-16 03:54:24 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" уже существует
Table "USERS" already exists; SQL statement:
CREATE TABLE users (name varchar(255), age int);  [42101-190]
2015-12-16 03:54:36 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Таблица "USERS" уже существует
Table "USERS" already exists; SQL statement:
CREATE TABLE users (name varchar(255), age int);  [42101-190]
2015-12-16 03:57:56 jdbc[6]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "DROP TABLE IF EXISTSUSERS[*]; "; ожидалось "EXISTS"
Syntax error in SQL statement "DROP TABLE IF EXISTSUSERS[*]; "; expected "EXISTS"; SQL statement:
DROP TABLE IF EXISTSusers; [42001-190]
2015-12-16 03:58:05 jdbc[6]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "DROP TABLE IF EXISTSUSERS[*]; "; ожидалось "EXISTS"
Syntax error in SQL statement "DROP TABLE IF EXISTSUSERS[*]; "; expected "EXISTS"; SQL statement:
DROP TABLE IF EXISTSusers; [42001-190]
2015-12-16 03:58:54 jdbc[6]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "DROP TABLE IF EXISTSUSERS[*]; "; ожидалось "EXISTS"
Syntax error in SQL statement "DROP TABLE IF EXISTSUSERS[*]; "; expected "EXISTS"; SQL statement:
DROP TABLE IF EXISTSusers; [42001-190]
2015-12-16 03:59:35 jdbc[6]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "DROP TABLE IF EXISTSUSERS[*]; "; ожидалось "EXISTS"
Syntax error in SQL statement "DROP TABLE IF EXISTSUSERS[*]; "; expected "EXISTS"; SQL statement:
DROP TABLE IF EXISTSusers; [42001-190]
2015-12-16 04:22:51 jdbc[6]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "DROP TABLE IF EXISTSUSERS[*]; "; ожидалось "EXISTS"
Syntax error in SQL statement "DROP TABLE IF EXISTSUSERS[*]; "; expected "EXISTS"; SQL statement:
DROP TABLE IF EXISTSusers; [42001-190]
2015-12-16 04:28:40 jdbc[6]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "DROP TABLE IF EXISTSUSERS[*]; "; ожидалось "EXISTS"
Syntax error in SQL statement "DROP TABLE IF EXISTSUSERS[*]; "; expected "EXISTS"; SQL statement:
DROP TABLE IF EXISTSusers; [42001-190]
2015-12-16 04:48:17 jdbc[6]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "DROP TABLE IF EXISTSUSERS[*]; "; ожидалось "EXISTS"
Syntax error in SQL statement "DROP TABLE IF EXISTSUSERS[*]; "; expected "EXISTS"; SQL statement:
DROP TABLE IF EXISTSusers; [42001-190]
2015-12-16 04:55:18 jdbc[6]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "DROP TABLE IF EXISTSUSERS[*]; "; ожидалось "EXISTS"
Syntax error in SQL statement "DROP TABLE IF EXISTSUSERS[*]; "; expected "EXISTS"; SQL statement:
DROP TABLE IF EXISTSusers; [42001-190]
